JMMServer/Providers/MyAnimeList/MALHelper.cs

```diff
@@ -530,15 +532,21 @@ public static void UpdateMALSeries(AnimeSeries ser)
                             ser.GetAnime().AnimeID);
                         continue;
                     }
+
+                    string confirmationMessage = "";
+                    if (animeListHashtable.ContainsKey(malID))
```

Comment:
@hidden4003 Actually, I think most things are back to normal, but I'm still having issues updating series. As a test, I removed some series from my MAL and adjusted some from Completed with votes to Watching with random numbers or 0 episodes watched with or without ratings. The log stated that they were successfully updated with the correct mal id, eps, and score, but my MAL had changes made to the series that I removed. Any series that I changed and needed to be updated using ModifyAnime() were once again not changed.